src/moscot/problems/space/_alignment.py

```diff
@@ -155,6 +171,14 @@ def solve(
             **kwargs,
         )  # type: ignore[return-value]
 
+    def _normalize_spatial(
```

Comment:
I think this can be defined in a new mixin class to avoid repeating the code (or alternatively in a utility function; have no strong opinion).
In the former case, how about using:
```python
def _normalize_spatial(self, normalize_key: Optional[str] = None) -> str:
   if normalize_key is None:
       return self.spatial_key
   spatial_key = f"{spatial_key}_{normalize_key}"
   self.adata.obsm[spatial_key] = ...
   return spatial_key
```
and later use it as:
```python
self.spatial_key = spatial_key
self.spatial_key = self._normalize_spatial(normalize_key)
```

Reply:
btw , in the `TranslationProblem` we expect the user to normalize the data beforehand. Can't we expect the user to do so?